mapdata/WarcraftLegacies/UnitData/Skin/1313026152.json

```diff
@@ -0,0 +1,58 @@
+{
+  "OldId": 1752458088,
+  "NewId": 1313026152,
+  "Unk": [
+    0
+  ],
+  "Modifications": [
+    {
+      "Id": 1718840949,
+      "Type": 3,
+      "Value": "(Frostwolf)"
+    },
+    {
+      "Id": 1953458293,
+      "Type": 3,
+      "Value": "W"
+    },
+    {
+      "Id": 1835101813,
+      "Type": 3,
+      "Value": "Frostwolf Commander"
```

Comment:
I wonder if we can come up with a better name than this? The faction is Frostwolf so prefixing Frostwolf just seems a bit redundant, and Commander feels like a pretty Western term for an Orc unit. Happy to brainstorm.

Reply:
yup sure, not my most inspired name. The thing is that the guy summons frostwolf. Maybe Frostwolf Packleader?

Reply:
Yup that's way better.